test/dataset/split/test_split.py

```diff
@@ -39,50 +39,150 @@ def test_ts_slice_to_item():
     sl.to_data_entry()
 
 
+def check_training_validation(
+    train_entry,
+    valid_pair,
+    prediction_length: int,
+    cutoff_date=None,
+    max_history=None,
+    split_offset=None,
+    window=1,
+    is_rolling_split=False,
+) -> None:
+    assert train_entry[FieldName.ITEM_ID] == valid_pair[0][FieldName.ITEM_ID]
+    if max_history is None:
+        assert len(train_entry[FieldName.TARGET]) == len(
+            valid_pair[0][FieldName.TARGET]
+        )
+    elif not is_rolling_split:
+        assert (
+            len(valid_pair[0][FieldName.TARGET]) + len(valid_pair[1])
+            == max_history
+        )
+    assert len(valid_pair[1]) == prediction_length * window
```

Comment:
Shouldn’t it just be 

```suggestion
    assert len(valid_pair[1]) == prediction_length
```